scroll-animations-1/Overview.bs

```diff
@@ -347,6 +347,57 @@ spec: cssom-view-1; type: dfn;
 	In the Web Animations API,
 	they can be represented anonymously by a {{ViewTimeline}} object.
 
+## View Progress Timeline Ranges ## {#view-timelines-ranges}
+
+	[=View progress timelines=] define the following [=named timeline ranges=]:
+
+	<dl dfn-for="animation-timeline-range" dfn-type="value">
+		<dt><dfn>cover</dfn>
+		<dd>
+			Represents the full range of the [=view progress timeline=]:
+			* 0% progress represents the position at which
+				the [=end=] [=border edge=] of the element’s [=principal box=]
+				coincides with the [=start=] edge of its [=view progress visibility range=].
```

Comment:
I think these are backwards. If the end edge of the element's box aligns with the start edge of the visibility range then the element has just finished scrolling past the scrollport (i.e. 100%). 0% should be when it just starts to scroll into the scrollport (i.e. the start edge of the principal box aligns with the end edge of the view progress visibility range.

Reply:
Depends on how you look at the “View Progress Visibility Range”. Does it go from top to bottom _(following the scrollport)_ or from bottom to top _(following the subject)_? Maybe this needs an explicit mentioned added to [the current definition](https://github.com/w3c/csswg-drafts/blob/ba5668843f6df246b832a562dc88d00da1df8461/scroll-animations-1/Overview.bs#L557)?

I would assume it to use the same names for its edges as the scrollport _(e.g. by default the bottom edge on the block axis would be named `end`)_

Reply:
Start/end are defined according to the writing mode of the scroll container, so I believe these are correct.